cgotorch/torch.cc

```diff
@@ -137,6 +137,15 @@ const char *Add(Tensor a, Tensor other, float alpha, Tensor *result) {
   }
 }
 
+const char *Add_(Tensor a, Tensor other, Tensor *result) {
+  try {
+    *result = new at::Tensor(a->add_(*other));
```

Comment:
To what I understand, here `a->add_` calls `aten::Tensor::add_`, which returns the reference to `a`, but not a newly created tensor. Am I correct?

If so, it seems that we don't need to call `SetTensorFinalizer` in the Go method `Tensor.AddInplace`?

https://github.com/wangkuiyi/gotorch/pull/158/files#diff-ae7d2f2494c2de4f64de14717ee14fbdR330